### ISSUE TITLE
List of all keyboard shortcuts

### ISSUE THREAD

**Issue by arteteco** [retext-project/retext#539]
Hi all, I'm having a hard time finding all the keyboard shortcuts, is there a menu voice for it? Can't find it in the docs neither.
Would be nice to have it in the preferences or at least in the Help menu

Thanks!

**Comment by mitya57** [comment]
Hi!

Unfortunately there is no central place where all shortcuts are documented, but most of them can be seen near the items in “File” and “Edit” menus. E.g. for me the Edit menu contents are:
```
Undo          Ctrl+Z
Redo          Ctrl+Shift+Z
----
Cut           Ctrl+X
Copy          Ctrl+C
Paste         Ctrl+V
Paste Image   Ctrl+Shift+V
....
```

Some shortcuts are defined by Qt and they are platform-dependent. See this table for a list: <https://doc.qt.io/qt-5/qkeysequence.html#standard-shortcuts>. ReText does not support all of them, but for example all related to editing text are supported.

**Comment by arteteco** [comment]
Thanks @mitya57 , yeah I knew about those. It would be nice to add them also to the Formatting menu, what do you think? I was looking for a shortcut to do a bullet list, or to turn a block of text into a code block... would be awesome to just have them, written on the side of the menu entry

**Comment by mrossw** [comment]
Hi !
I was wondering if there is a shortcut to indent a line where the cursor is. Just like  Shift+<tab> works to unident a line.

**Comment by mitya57** [comment]
No, there is no such shortcut. But it's quite easy to use <kbd>Home</kbd> and then <kbd>Tab</kbd>.
